### PR TITLE
feat(UI-Toolkit) - Make toggles to spec

### PR DESCRIPTION
**Pull request by SaraVieira**

### CODE REVIEW

File: packages/ui-toolkit/src/form/toggle.js [hunk 1]
```diff
@@ -73,24 +77,49 @@ const InputContainer = styled.div`
...
+  position: relative;
+  margin-top: ${unitcalc(1)};
+
+  li:first-of-type {
+    input + label + .background {
+      transform: translateX(calc(100% - 3px));
+    }
+
+    input:checked + label + .background {
+      transform: translateX(-1px);
```
Comment (ramitos) [inline packages/ui-toolkit/src/form/toggle.js]: rem?

File: packages/ui-toolkit/src/form/toggle.js [hunk 2]
```diff
@@ -73,24 +77,49 @@ const InputContainer = styled.div`
...
+  align-items: flex-end;
+  background-color: ${props => props.theme.disabled};
+  border-radius: ${borderRadius};
+  border: ${border.unchecked};
+  position: relative;
+  margin-top: ${unitcalc(1)};
+
+  li:first-of-type {
+    input + label + .background {
+      transform: translateX(calc(100% - 3px));
```
Comment (ramitos) [inline packages/ui-toolkit/src/form/toggle.js]: also, rem?

File: packages/ui-toolkit/src/form/toggle.js [hunk 3]
```diff
@@ -73,24 +77,49 @@ const InputContainer = styled.div`
...
+  background-color: ${props => props.theme.white};
+  display: block;
+  border-radius: ${borderRadius};
+  position: absolute;
+  height: 100%;
+  width: 100%;
+  z-index: 0;
+  transition: all 200ms ease-out;
+  border: 0.0625rem solid #d8d8d8;
+  top: -1px;
```
Comment (ramitos) [inline packages/ui-toolkit/src/form/toggle.js]: rem?

File: packages/ui-toolkit/src/form/toggle.js [hunk 4]
```diff
@@ -73,24 +77,49 @@ const InputContainer = styled.div`
...
+  border-right: ${border.unchecked};
+  background-color: ${props => props.theme.white};
+  display: block;
+  border-radius: ${borderRadius};
+  position: absolute;
+  height: 100%;
+  width: 100%;
+  z-index: 0;
+  transition: all 200ms ease-out;
+  border: 0.0625rem solid #d8d8d8;
```
Comment (ramitos) [inline packages/ui-toolkit/src/form/toggle.js]: remcalc?

File: packages/ui-toolkit/src/form/toggle.js (hunk 1 above)
Comment (SaraVieira) [reply packages/ui-toolkit/src/form/toggle.js]: I like how you don't condemn my hack but tell me to use rem 😝

Comment (ramitos) [reply packages/ui-toolkit/src/form/toggle.js]: I'm all for clever hacks 😄

**Pull request merged**